package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stremio-addon-sdk",
-  "version": "0.6.4",
+  "version": "0.7.1",
```

Comment:
let's use `0.6.5` as this is not a major change and won't break any current usage of the Add-on SDK